### PR TITLE
Integrate Slate docs in new documentation

### PR DESCRIPTION
**Pull request by lemieux**
This pretty much strips out the non-REST documentation from the slate docs and adds build steps to publish them at the right place.

@alavers @mspensieri @dannytranlx

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -3,11 +3,17 @@ machine:
...
+    version: 2.3.0
 
 dependencies:
   override:
     - npm install
     - sudo pip install awscli
+    - cd slate
+    - gem install bundler
+    - gem install middleman
+    - npm install
```
Comment (dannytranlx) [inline circle.yml]: `npm install` is there twice?

Comment (lemieux) [inline circle.yml]: one first the gatsby project, one for the slate project

Comment (lemieux) [inline circle.yml]: but I think it didn't work

Comment (lemieux) [inline circle.yml]: yeah but I think it doesn't stay in that folder.. I'll just add it to every line

**Pull request merged**